### ISSUE TITLE
Having a functional tensorRT extension seems to mean not having access to the clip interogate feature?

### ISSUE THREAD

**Issue by left1000** [NVIDIA/Stable-Diffusion-WebUI-TensorRT#126]
See https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/13689 it seems like maybe people just gave up? Is there no way to have clip interrogate working at the same time tensorRT is installed?

**Comment by contentis** [comment]
As I'm not aware of this feature, could you elaborate on what exactly clip interrogate does (ideally from a technical perspective)? 
Or can you provide instructions on how to set it up and reproduce this error?

**Comment by left1000** [comment]
"Interrogate clip" is a default feature for img2img in automatic1111webgui. As far as I know it's always been there and always been functional.

It's never not been set up, and I'm not sure what steps I might've taken to break it, other than getting tensorRT working properly. Both the timing, and the other issue I linked are my only evidence.

This is the error message.

<img width="1280" alt="image" src="https://github.com/NVIDIA/Stable-Diffusion-WebUI-TensorRT/assets/16432525/40e74178-bf0f-4299-a60b-ef41be6c0474">

I don't know much about how to install clip interrogate, because like I said, it's a default feature of automatic1111webgui that has always worked without me ever having had to do anything.

I tried setting if setting the sd_unet back to none 
<img width="1270" alt="image" src="https://github.com/NVIDIA/Stable-Diffusion-WebUI-TensorRT/assets/16432525/c94f9d8e-c999-43c9-aff9-73030b0c557b">

Was related or would fix clip interogate. It did not help. It did however trigger this bug: which is too vague to explain https://github.com/AUTOMATIC1111/stable-diffusion-webui-tensorrt/issues/69

In essence any time I do anything at all except using this extension properly, all/some of my profiles are set to invalid. So merely unselecting the sd_unet in the webgui means I'll have to rebuild all of my engine profiles. If I could use force rebuild to rebuild just the broken one(s) that wouldn't be so bad. But I have to rebuild them all. I think somehow force rebuild is not effecting models.json properly as I mentioned here https://github.com/AUTOMATIC1111/stable-diffusion-webui-tensorrt/issues/73

**Comment by left1000** [comment]
I created this thread to ask how someone as inept as me might update to the dev branch https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/13760 if anyone reading this issue knows how.

I figured I didn't need to be on the dev branch because the instructions I first read about his extension telling me to go to the dev branch were two months old. So I figured the master branch, which update.bat is pulling it's git pull from, has probably caught up by now. But that's just a wild guess by me :)

**Comment by left1000** [comment]
Okay so my issues have nothing at all to do with dev branch or master branch, as it seems automatic1111's dev branch and master branch happen to be identical today?

<img width="416" alt="image" src="https://github.com/NVIDIA/Stable-Diffusion-WebUI-TensorRT/assets/16432525/99de11f3-8a61-4769-a0cb-3f7c6f51514d">

**Comment by chazzhou** [comment]
I can replicate this issue. Simply rename `models.py` under this extension to `models_trt.py`, 
and change line 12 of `ui_trt.py` from
`from models_trt import make_OAIUNetXL, make_OAIUNet`
to
`from models import make_OAIUNetXL, make_OAIUNet`

See #121

**Comment by left1000** [comment]
> models_trt

Thank you so much, this fix absolutely works. And better yet, it helps me feel like I'm not losing my mind. I'm so new to this ai stuff I thought maybe it was 100% my own ineptness causing me to see imaginary problems everywhere :D

TLDR: I made your changes and it works. I can confirm rt rtensor and clip interrogate are working on my machine both.

**Comment by left1000** [comment]
> models_trt

Thank you so much, this fix absolutely works. And better yet, it helps me feel like I'm not losing my mind. I'm so new to this ai stuff I thought maybe it was 100% my own ineptness causing me to see imaginary problems everywhere :D

TLDR: I made your changes and it works. I can confirm rt rtensor and clip interrogate are working on my machine both.

**Comment by chazzhou** [comment]
@left1000 I'm glad it works! I suggest keeping the issue open until the pull request is merged so the maintainer and others can see it. Have a great day!

**Comment by left1000** [comment]
I only know how to close or open an issue when inputting a new comment. So to follow up with your suggestion... I typed this sentence.
